molecule/default/tests/test_verify_users_and_tenants.py

```diff
@@ -0,0 +1,42 @@
+import os
+import re
+import pytest
+import testinfra.utils.ansible_runner
+
+#######################################################################
+# ASC-236: Verify the requested user(s) and tenant(s) were created    #
+# and assigned the proper role.                                       #
+#                                                                     #
+# Notes: These tests should be in keystone submodule, however, there  #
+# are not many test cases for keystone at this time so we are putting #
+# them here in novo submodule. When the keystone test cases grow in   #
+# the future, we should create keystone submodule for them            #
+#######################################################################
+# RPC 10+ manual test 7
```

Comment:
Instead of using a comment block, utilize doc strings:

```
"""ASC-236: Verify the requested user(s) and tenant(s) were created    #
and assigned the proper role.
...
"""